sagemathinc/cocalc
# latex: fix #3111 edge case

see #3111 

what I did to test this:

* scenario A (sanitization)
  1. vim-edit the `.foo.tex.syncdb` file to have `""` as the build command
  2. open the tex file
  3. check that it builds and the build command in syncdb is the default

* scenario B (changes)
  1. either vim-edit the syncdb file or open up the page twice
  2. changes on one instance properly show up on the other side and vice versa
